backend/backend-python/model_loader.py

```
@@ -132,9 +164,12 @@ def load_model(
                 loaded_keys.add(target_name)
 
         if "lm_head.weight" in model_state_keys and "lm_head.weight" not in loaded_keys:
-            model.state_dict()["lm_head.weight"].copy_(
-                model.model.embed_tokens.weight, non_blocking=True
-            )
+            if hasattr(model, "model") and hasattr(model.model, "embed_tokens"):
+                embed_tokens = getattr(model.model, "embed_tokens")
```

Ruff (0.13.1)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)